### ISSUE TITLE
New Project

### ISSUE THREAD

**Issue by LostSoulFly** [LostSoulFly/MinerProxy#23]
sneaky

**Comment by LostSoulFly** [comment]
@samut3 I couldn't find your email address, so I just closed a new issue.

Are you interested in working with me on a new project related to crypto?

I have a couple things in mind, the biggest two are a re-write of MinerProxy and a MinerManager of sorts.

For MinerProxy, RajanGrewal did some updates to the original proxy (some changes he borrowed from us, funny enough), but he did some fixes as well. I'd like to add better failover as well as make it easier to add features. A nice cleanup, basically.

For MinerManager, I'd like to make a client/server that allows for managing all of your miners from one program.

For MinerManager, I'm thinking we could make a free version and a paid version. You'd of course get a cut of the profits if it works out, and we'd take payment in crypto to keep it simple. I think some basic features can be free, or we can limit the free version to a couple miners. Maybe only allow encryption between server/client on the paid version.

I've got a lot of ideas I'd like to implement, but the basics would be:

Easy to see which miner is mining what, as well as their current hashes and perhaps a performance history.
Easily switch miners to different coins with a click, like switching from ETH to Monero and changing mining software automatically.
Monitoring miners status and uptime, as well as temperatures and health of GPUs/CPUs.
Easy to configure computers for mining, such as cleaning up temp files and restoring lost space or running various scripts to improve performance.
Control miners like shitting down, rebooting, etc. Also schedule reboots from the server, or restart if issues happen.
Telegram notifications and commands (probably a paid version perk, there are libraries for this and it's super easy to implement) such as uptime, stats, and managing miners.

Some other stuff I'd like to do with it:

Remote desktop into the servers to control them if RDP or whatever isn't working.
Scripting, using a c# scripting language and include the ability to pull current most profitable coins from somewhere (there are some c# libraries that already support this). Automatically perform profitability switching, basically.
Wallet management and monitoring their current coins and payouts
Include a version of MinerProxy that automatically proxies information and rig stats (may not be necessary)

I've already got an idea of which networking library I'd like to use, but I haven't started work on it as of yet.
